src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/OrderlyServerAPI.kt

```diff
@@ -1,14 +1,18 @@
 package org.vaccineimpact.orderlyweb
 
 import khttp.responses.Response
+import org.json.JSONArray
+import org.json.JSONObject
 import org.vaccineimpact.orderlyweb.db.Config
 
 interface OrderlyServerAPI
 {
-    fun post(url: String, context: ActionContext): Response
-    fun get(url: String, context: ActionContext): Response
+    fun post(url: String, context: ActionContext): OrderlyServerResponse
+    fun get(url: String, context: ActionContext): OrderlyServerResponse
 }
 
+data class OrderlyServerResponse(val text: String, val statusCode: Int)
```

Comment:
The standard Response objects from the khttp package were previously being passed straight through to the caller, but they aren't mutable, so I've just made a little class to hold the only bits of Response which were actually used to modify OrderlyWeb's own response.